src/modules/service-catalog/components/service-catalog-list-item/ServiceCatalogListItem.tsx

```diff
@@ -0,0 +1,88 @@
+import type { ServiceCatalogItem } from "../../data-types/ServiceCatalogItem";
+import ShapesIcon from "@zendeskgarden/svg-icons/src/16/shapes-fill.svg";
+import styled from "styled-components";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+
+const StyledItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
```

Comment:
Why do we need this?

Reply:
Oh right maybe we don't need that much styles now. It must have stayed after my attempt to customize Tiles.